pkg/kubelet/kubelet_pods.go

```diff
@@ -729,7 +729,7 @@ func (kl *Kubelet) getPullSecretsForPod(pod *v1.Pod) []v1.Secret {
 	return pullSecrets
 }
 
-// Returns true if pod is in the terminated state ("Failed" or "Succeeded").
+// podIsTerminated returns true if pod is in the terminated state ("Failed", "Succeeded" or "DeletionTimestamp" not nil).
```

Comment:
The new comment is not accurate. This method return true when "DeletionTimestamp" is not nil AND pod containers are not running.